### PR TITLE
[Sprint Maintain]Feature create ui quick order

### PR DESCRIPTION
**Pull request by nguyenhuy95dn**
Feature create ui quick order
Link redmine : https://dev.framgia.com/redmine/issues/123089
Task : Quick Order
Effect: N/A

### CODE REVIEW

File: app/src/main/java/com/framgia/forder/screen/quickorder/QuickOrderViewModel.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        return String.format(Utils.FORMAT_PRICE, mPrice * mQuantity) + UNIT_MONEY;
+    }
+
+    @Bindable
+    public String getProductNumber() {
+        return String.valueOf(mQuantity);
+    }
+
+    public void downQuantity() {
+        if (mQuantity == 1) {
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/quickorder/QuickOrderViewModel.java]: pls define ``1`` as a constant variable

File: app/src/main/res/drawable/border_button_order.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <gradient
+        android:angle="270"
+        android:centerColor="@color/color_red_light"
+        android:endColor="@color/color_red_light"
+        android:startColor="@color/color_red_light"/>
+    <corners android:radius="@dimen/dp_10"/>
+</shape>
```
Comment (daolq3012) [inline app/src/main/res/drawable/border_button_order.xml]: enter when end of file

File: app/src/main/res/drawable/border_edittext.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        <shape android:shape="rectangle">
+            <solid android:color="@color/color_white"/>
+            <corners android:radius="@dimen/dp_5" />
+            <stroke
+                android:width="@dimen/dp_1"
+                android:color="@color/color_gray"
+                />
+        </shape>
+    </item>
+</selector>
```
Comment (daolq3012) [inline app/src/main/res/drawable/border_edittext.xml]: like above

File: app/src/main/res/drawable/border_button_order.xml (hunk 2 above)
Comment (nguyenhuy95dn) [inline app/src/main/res/drawable/border_button_order.xml]: added

File: app/src/main/res/drawable/border_edittext.xml (hunk 3 above)
Comment (nguyenhuy95dn) [inline app/src/main/res/drawable/border_edittext.xml]: added

**Pull request merged**